tests/integration/features.go

```diff
@@ -413,9 +436,6 @@ func GetTracesFeature() features.Feature {
 				// "_sourceCategory":    "kubernetes/customer/trace/tester/customer/trace/tester",
 				"_sourceName": fmt.Sprintf("%s.%s.%s", internal.TracesGeneratorNamespace, internal.TracesGeneratorName, internal.TracesGeneratorName),
 			},
-			internal.ReceiverMockNamespace,
-			internal.ReceiverMockServiceName,
-			internal.ReceiverMockServicePort,
```

Comment:
We are not going to check receiver-mock metrics anymore?

Reply:
This doesn't change any of the test checks, I just moved this logic to a separate function which gets these values from module constants.

Reply:
I do want to get rid of checking for receiver-mock metrics, because they're annoying to work with - receiver-mock only emits some of them based on ingested data, so they're different for each test - but I didn't do that in this PR.